.github/workflows/nestbuildmatrix.yml

```diff
@@ -687,7 +681,6 @@ jobs:
           make install
         env:
           xNEST_BUILD_TYPE: ${{ matrix.NEST_BUILD_TYPE }}
```

Comment:
If this is removed from the matrix, the empty variable can probably go away here too.
```suggestion
```

Reply:
Here I am out of my depth. What do you mean by "empty variable"? And could we then remove the entire `env:` section? And where is `NEST_BUILD_TYPE` actually set?

Reply:
empty meaning it is not set and therefore results in an empty string being set as `xNEST_BUILD_TYPE` variable for the script.
The expression `matrix.NEST_BUILD_TYPE` refers to the `strategy:matrix:` defining the jobs, not a shell variable. The `env:` section would transfer this into a `xNEST_BUILD_TYPE`-named shell environment variable.
If it's empty, the `env:` can be dropped completely.